src/utils/uploader.js

```diff
@@ -112,18 +112,35 @@ export class BlobUploader {
         tx.blobVersionedHashes = versionedHashes;
         tx.blobs = ethersBlobs;
         tx.kzg = kzg;
-        return await this.#wallet.sendTransaction(tx);
+        return tx;
     }
 
-    async sendTxLock(tx, blobs) {
+    async sendTx(tx, blobs) {
```

Comment:
What is the use case of sendTx instead of  sendTxLock?

Reply:
Synchronization locks will only be executed for concurrent transactions, and EthStorage ‘write’ are not concurrent.